env/numerical_compliance.txt

```diff
@@ -436,6 +456,11 @@ devices given as ULP values.
 | 0 ulp
 | 0 ulp
 
+| *OpExtInst* *normalize*
+| \<= 1.5 + (0.5 * n) + (0.5 * (n - 1)) ulp, for gentype with vector width n
```

Comment:
First value here represents error in rsqrt + error in multiply. The fp16 extension defines rsqrt as having ulp of 1 and multiply as being correctly rounded